test/performance-tests.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script runs the presubmit tests; it is started by prow for each PR.
+# For convenience, it can also be executed manually.
+# Running the script without parameters, or with the --all-tests
+# flag, causes all tests to be executed, in the right order.
+# Use the flags --build-tests, --unit-tests and --integration-tests
+# to run a specific set of tests.
+
+source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/e2e-tests.sh
+
+# Location of istio for the test cluster
+readonly ISTIO_YAML=./third_party/istio-1.0.1/istio-lean.yaml
+
+function create_istio() {
```

Comment:
duplicated code, e2e tests have the same for sharing.

Reply:
yeah. I cannot source e2e-tests.sh as it will run the e2e tests, which I do not want. Maybe this should be moved to test-infra scripts?

Reply:
since lots of code can be shared, do we really need a separate perf script, can we just use e2e-tests script with parameters?

Reply:
Definitely this is duplicated code that should be in a single place, and not in test-infra. I suggest creating a helper script that will be sourced by serving's `e2e-tests.sh` and this script. Such helper will contain all the functions for bringing up or down the cluster.

Reply:
created a new file for cluster operations